renovate.json

```diff
@@ -1,10 +0,0 @@
-{
-  "extends": ["config:js-lib"],
-  "automerge": false,
-  "patch": {
-    "automerge": true
-  },
-  "devDependencies": {
-    "automerge": true
-  }
-}
```

Comment:
I left it out because I wasn't completely sure about how it interacted with the library, but I've just reintroduced it. Short answer: file removed due to my uncertainty and naiveté. 😅